src/compute.ts

```diff
@@ -72,11 +101,12 @@ function ec2VolumeInUseCheck(enforcementLevel: EnforcementLevel = "advisory", ch
     };
 }
 
-export function elbAccessLoggingEnabled(enforcementLevel: EnforcementLevel = "advisory"): ResourceValidationPolicy {
+/** @internal */
+export function elbAccessLoggingEnabled(enforcementLevel?: EnforcementLevel): ResourceValidationPolicy {
     return {
         name: "elb-logging-enabled",
         description: "Checks whether the Application Load Balancers and the Classic Load Balancers have logging enabled.",
-        enforcementLevel: enforcementLevel,
+        enforcementLevel: enforcementLevel || defaultEnforcementLevel,
```

Comment:
This `enforcementLevel || defaultEnforcementLevel` pattern seems kinda tedious. Couldn't we have the default enforcement level be checked/passed by whatever function ends up calling the `ResourceValidationPolicy`?

Reply:
Originally, I had an overload of `getValueOrDefault` and the implementation handled this case, but it actually made the implementation of `getValueOrDefault` more difficult to understand, so I opted to just go with the simpler `enforcementLevel || defaultEnforcementLevel`. Changing the caller to pass the right enforcement level (whether one was specified or the default) would involve changing the signature of the function to not be an optional parameter, but I'd rather keep it optional to match the associated property on the `AwsGuardArgs` interface.